edenia/antelope-tools
# Redesign nodes page 1304

### Redesign nodes page and apply styles adjustments for BPs pages

### What does this PR do?

- Resolve #1304
- Resolve #1287
- Resolve #1235
- Apply adjustments in mobile version
- Avoid to request eosRate stats if there is no an available link
- Refactor folders related with the block producers comoponets
- Use the theme to set some colors

### Steps to test

1. Run the project locally
2. Go to nodes page
3. Check the changes on desktop and mobile version
4. Go to any BP Profile
5. Check the changes

### Screenshots

# Nodes

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/aa614dec-3ad9-4fff-9812-8a21f108b016)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/ec14b908-d226-45ee-b52c-6317a23327c5)

# BP Profile

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/d5e84f5e-ce34-4ffd-9fee-2b4c22c6ab33)

# Block Producers List

![imagen](https://github.com/edenia/antelope-tools/assets/66583677/d64c2d40-8a4b-48eb-8d42-c9bcc8dcfbc2)
![imagen](https://github.com/edenia/antelope-tools/assets/66583677/02143a3d-08e7-43fa-8239-4159c7d622e5)
